noir-projects/noir-contracts/contracts/private_fpc_contract/src/main.nr

```diff
@@ -20,19 +20,19 @@ contract PrivateFPC {
     }
 
     #[aztec(private)]
-    fn fund_transaction_privately(amount: Field, asset: AztecAddress, nonce: Field) {
+    fn fund_transaction_privately(amount: Field, asset: AztecAddress, randomness: Field) {
         assert(asset == storage.other_asset.read_private());
         // convince the FPC we are not cheating
-        context.push_nullifier(nonce, 0);
+        context.push_nullifier(randomness, 0);
 
         // allow the FPC to reconstruct their fee note
-        emit_nonce_as_unencrypted_log(&mut context, nonce);
+        emit_randomness_as_unencrypted_log(&mut context, randomness);
```

Comment:
This is yet very unfinished (see [here](https://github.com/AztecProtocol/aztec-packages/pull/7226/files#r1664154821)) so I will document it once it's more finalized.